module/modmanager/manager.go

```diff
@@ -841,6 +905,21 @@ func (m *module) cleanupAfterStartupFailure(mgr *Manager, afterCrash bool) {
 	}
 }
 
+func (m *module) getFullEnvironment(viamHomeDir string) map[string]string {
+	environment := map[string]string{
+		"VIAM_HOME":        viamHomeDir,
+		"VIAM_MODULE_DATA": m.dataDir,
+	}
+	if m.cfg.Type == config.ModuleTypeRegistry {
+		environment["VIAM_MODULE_ID"] = m.cfg.ModuleID
```

Comment:
Aside but with all these env variables, this makes me wish we did module log levels as environment variables as @Otterverse originally suggested. At the least, would unify logic here instead of having all of this _and_ a command line argument for log level.

Reply:
*doesn't say it* :-)